### ISSUE TITLE
Release `wasi-messaging@0.2.0`

### ISSUE THREAD

**Issue by danbugs** [WebAssembly/wasi-messaging#26]

**Comment by thomastaylor312** [comment]
Just noting that this is going to be dependent on #24. We may also want to do some RCs first

**Comment by ricochet** [comment]
Yes let's have rc's and then for phase 3 is the time to cut `0.2.+`. If we're adding to WASIP2 then I also think we would want to match the latest wasi 0.2.+ versions.
